bradybridges/jetcake-react-app
# Profile

#### What's this PR do?
Adds toggle modal style for login and create account. Adds functionality to login. Adds functionality to store profile information in the cloud. Retrieves profile information on login. Adds basic create account and sign in forms. Adds ability to edit profile information. Updates testing for new components and additional methods.
#### How should this be manually tested?
Run `npm test` and make sure all tests are passing. Verify that you can create an account, signout, login, etc. When signed in you should be able to view profile page.
#### Any background context you want to provide?
Haven't added much styling will need to touch up UI once functionality is completed.
#### What are the relevant tickets?
#14 #15  #8 #9 #11 #12 
#### Screenshots (if appropriate)
![profile](https://user-images.githubusercontent.com/25031031/73494872-9f875280-4372-11ea-9670-7b4c1e8f3a78.PNG)
![editProfile](https://user-images.githubusercontent.com/25031031/73494875-a0b87f80-4372-11ea-8dee-7cccb3002da8.PNG)
